manifests/filter/open_tracing.pp

```diff
@@ -0,0 +1,180 @@
+# == Resource:  repose::filter::open_tracing
+#
+# This is a resource for enabling emitting traces to an OpenTracing tracer (current supports Jaeger).
+# Only usable in >= 8.8.3.0
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename.
+# Defaults to <tt>open-tracing.cfg.xml</tt>
+#
+# [*service_name*]
+# String. Application name
+# Defaults to <tt>repose</tt>
+#
+# [*connection_http*]
+# Object that sets up http connection.  Contains endpoint, username, password, and token
+# Defaults to <tt>undef</tt>
+#
+# [*endpoint*]
+# String. Set to http endpoint to push traces to.  Required if using http connection.
+# Defaults to <tt>undef</tt>
+#
+# [*username*]
+# String. Set if http endpoint requires username/password authentication.  Must be used in conjunction with password
+# Defaults to <tt>undef</tt>
+#
+# [*password*]
+# String. Set if http endpoint requires username/password authentication.  Must be used in conjunction with username
+# Defaults to <tt>undef</tt>
+#
+# [*token*]
+# String. Set if http endpoint requires token authentication.
+# Defaults to <tt>undef</tt>
+#
+# [*connection_udp*]
+# Object that sets up udp connection.  Contains host and port.
+# Defaults to <tt>undef</tt>
+#
+# [*port*]
+# Integer. Set to port the agent is listening on.
+# Defaults to <tt>undef</tt>
+#
+# [*host*]
+# String. Set to host the agent is listening on.
+# Defaults to <tt>undef</tt>
+#
+# [*sampling_constant*]
```

Comment:
if the only subparameter for sampling_constant is toggle, and it is yes or no, why not just make the top level parameter such as `enable_sampling` or even just change this to `sampling_constant_toggle`